code/game/machinery/OpTable.dm

```diff
@@ -30,6 +31,8 @@
 		computer = null
 	if(victim)
 		victim = null
+	injected_reagents.Cut()

```

Comment:
Well, yes, but I'm pretty sure that gets cleared when the object soft garbage collects or hard deletes.

In either case, that *shouldn't* be in memory anymore; the only time this becomes a problem is when the list contains a reference to an object that still exists; then it will fail to soft GC....even if it fails, however, 30 seconds later it will be hard deleted and it will, for sure, clear this list.

@Krausus my reasoning/logic is correct, yes?